apps/site/src/content/blog/state-of-the-budgie-2023.mdx

```diff
@@ -0,0 +1,201 @@
+
+- Show Desktop Applet has been ported
+- Tab Switcher has been ported
+- Workspaces Applet has been ported
+- IconTasklist Applet is currently being ported.
+
+Keep in mind that not every part of the codebase needs to be touched, but some key areas in the codebase will need to be updated once some newer window tracking logic has been merged in from the IconTasklist conversion: MenuManager (right-click menus), Panel Manager (manages panels), the legacy Tasklist needs to be rewritten, and various uses of Gdk.X11 APIs.
+
+On the compositor front, the progress by Campbell Jones (aka serebit) has been fantastic as well. Budgie's Wayland compositor, codenamed Magpie v1, was started in the form of a separate project called Flyaway at the end of April 2023. Since then, Magpie v1 has been fleshed out into a reasonably functional window manager with support for things X11 could only dream of. It's still a ways from becoming ready to integrate into Budgie proper, but you can already use it to run Waydroid and first-person Steam games (though variable refresh rate and direct scanout support are still pending)!
+
+## 2024
+
+For the new year, our focus is on two "epics".
+
+### Budgie 10 Epic
+
+Our main epic is delivering Budgie 10 on Wayland. This does not necessarily mean everything will be feature complete, as we still need to finesse, research and define what else needs to be done. Our aim though is to ensure our first release can be daily driven and consumed.
+
+For Q1, our focus will be:
+
+1. Delivering Budgie 10.9 (as early as this month) featuring our initial work with libxfce4windowing, as well as our new requirement for using budgie-session for launching Budgie Desktop. This will include the existing applets and Tab Switcher that have been ported and should provide plenty of time for distribution packagers to get Budgie Desktop into their respective releases without running up against any freeze deadlines.
+2. Expand on various definitions of work for the year related to Budgie 10 in the areas of: daemon and xdg / flatpak portal.
+3. Getting Budgie Desktop running in a Wayland or XWayland session for better testing.
+4. Early daemon development work that was deferred last year. Some of this work will be done for the purposes of bridging the existing Budgie Control Center and Magpie v1.
+
+For Q2, our goals are:
+
+1. Round out any remaining Wayland work in Budgie Control Center and Budgie Desktop specifically so it should be usable under Magpie v1.
+2. Begin any portal work defined in Q1.
+
+For Q3 and Q4, our focus is to deliver a Budgie 10 release that is Wayland-only and **should** we consider the experience ready to be daily driven for the wider community, we will officially put Budgie 10 into maintenance mode. Maintenance mode for us means that only changes to facilitate Wayland support and bug fixes would be addressed. This would be addressed by a dedicated Budgie 10 team.
```

Comment:
The first sentence has a run-on clause.

Reply:
Should be resolved with 055e4734eb9e38b2848c95cbf82af23ceb42c99f

Care to take a look at it again?

Reply:
LGTM!